GCSViews/ConfigurationView/ConfigUAVCAN.Designer.cs

```diff
@@ -610,7 +610,7 @@ private void InitializeComponent()
         private System.Windows.Forms.DataGridViewTextBoxColumn Node;
         private System.Windows.Forms.DataGridViewTextBoxColumn Level;
         private System.Windows.Forms.DataGridViewTextBoxColumn Source;
-        private System.Windows.Forms.DataGridViewTextBoxColumn Text;
+        private new System.Windows.Forms.DataGridViewTextBoxColumn Text;

```

Comment:
this one shouldnt be here

Reply:
I took care of the VS helpers.
Does this mean that this support is unnecessary?